samples/wifi/wfa_qt_app/README.rst

```diff
@@ -180,6 +180,190 @@ See `QuickTrack Test Tool Platform Integration Guide`_ for the installation proc
 
 See `Platform Intel Ax210 Integration Guide`_ for more details.
 
+QuickTrack Configuration settings
+=================================
+
+This section provides information on setting up the Quicktrack test tool.
+
+Settings
+--------
+
+The diagram below provides an overview of the test setup configuration settings.
+
+.. figure:: /images/test_setup_configuration.png
+     :width: 780px
+     :align: center
+     :alt: Wi-Fi QuickTrack test setup configuration
+
+     Test Setup Configuration
+
+Tool Platform
+^^^^^^^^^^^^^
+
+You can adjust the Tool Platform settings by providing the following information:
+
+* **Tool Platform IP Address**: Ethernet IP address of the QuickTrack Test Tool platform.
+* **Tool Platform Port**: Port number on which the ControlAppC service is running on the
+QuickTrack Test Tool platform.
+* **Control Interface Port**: It is usually left at its default setting.
+* **Platform Wireless IP Address**: Wireless IP Address of the QuickTrack Test Tool platform.
+
+Device Under Test (DUT)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+You can adjust the DUT settings by providing the following information:
+
+* **DUT Type**: It must be set to ``Station`` for STAUT and ``Access Point`` for APUT.
+* **DUT Control IP Address**: Ethernet IP address of the DUT (It can be either a netUSB or SLIP IP).
+* **DUT Port**: Port number on which ControlAppC service is running on the DUT.
+* **DUT Wireless IP Address**: The wireless IP Address of the DUT, which is in the same subnet as
+the Platform Wireless IP Address.
```

Comment:
```suggestion
* **DUT Type**: It must be set to ``Station`` for STAUT and ``Access Point`` for APUT.
* **DUT Control IP Address**: Ethernet IP address of the DUT (It can be either a netUSB or SLIP IP).
* **DUT Port**: Port number on which ControlAppC service is running on the DUT.
* **DUT Wireless IP Address**: The wireless IP Address of the DUT, which is in the same subnet as the Platform Wireless IP Address.
```